### ISSUE TITLE
a bug of cdp.Text

### ISSUE THREAD

**Issue by stephencheng** [chromedp/chromedp#137]
@rjeczalik @kenshaw @pwaller 

I experienced a problem with random in consistence of grab data text, and I am not sure where the bug is or relating to applying @rjeczalik 's fix.

```
	err = c.Run(ctxt, cdp.Tasks{
		cdp.Click(`#arefreshlink`, cdp.NodeVisible),
		cdp.WaitVisible(`#alinkselector`),
		cdp.Text(sel1, &text1),
		cdp.Text(sel2, &text2),
		cdp.Text(sel3, &text3),
                .......
		cdp.Text(sel16, &text16),

	})

```

I make a for loop for 20 times, 1 in 5, or 20% of the chance is that 1 or 2 fields will be empty. However this is not the case for the web page, the page does show the text correctly.

To ensure that there is no data source error, I use casperjs to have tested same case and it never fail and actually I used it in production case, it just does not fail at all to get an empty text from chrome side.

To ensure the test is 100% accurate, I did two things:

1. I use a timeout retry, say in 20 seconds to try to poll every second to see if the element's text will be refilled if it's due to async timing issue. This however didn't rescue and obtain the text value

2.  I use a function call to get the block of thos fields, like sel1 ... sel16 by getting the parent node's text: cdp.OuterHTML(sel, &htm_text),

this function call was put a step prior to the retry block and it always get correct non-empty value of the text.

In conclusion, I'd say this is a genuine bug indeed, please could you look into it if possible

**Comment by mvdan** [comment]
Very likely a duplicate of https://github.com/chromedp/chromedp/issues/168; let's keep the discussion there.
